cypress-io/cypress
# chore(deps): migrate to community @faker-js/faker

- Closes https://github.com/cypress-io/cypress/issues/30096

### Additional details

- The npm module [faker](https://www.npmjs.com/package/faker) is currently listed as deprecated on https://github.com/cypress-io/cypress/issues/3777

The deprecated npm module [faker@5.5.3](https://www.npmjs.com/package/faker/v/5.5.3) is migrated to the identical (non-deprecated) community version [@faker-js/faker@5.5.3](https://github.com/faker-js/faker/releases/tag/v5.5.3). See note in https://v6.fakerjs.dev/migration-guide-v5/

> In addition to releasing all future versions under the @faker-js/faker package namespace, we have also provided all historical versions of Faker.

For background information read also [I heard something happened. What's the TLDR?](https://fakerjs.dev/about/announcements/2022-01-14.html#i-heard-something-happened-what-s-the-tldr).

### Steps to test

```shell
git clean -xfd
yarn
yarn workspace @packages/frontend-shared cypress:run:ct
yarn workspace @packages/frontend-shared check-ts
yarn workspace @packages/app check-ts
```

### How has the user experience changed?

No change.

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
